backend/dal/Migrations/20210126015055_v01.06.03.cs

```diff
@@ -42,9 +46,14 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 columns: new[] { "Id", "IsSensitive", "AgencyId", "ClassificationId", "PID", "PIN", "AddressId", "LandArea", "Zoning", "ZoningPotential" });
 
             migrationBuilder.CreateIndex(
-                name: "IX_Buildings_Id_IsSensitive_AgencyId_ClassificationId_AddressId_ProjectNumbers_BuildingConstructionTypeId_BuildingPredominateUs~",
+                name: "IX_Buildings_Id_IsSensitive_AgencyId_ClassificationId_AddressId_ProjectNumber_BuildingConstructionTypeId_BuildingPredominateUse~",
```

Comment:
I would have thought it should be named `...ProjectNumbers...` not `...ProjectNumber...`?